test/correctness/gpu_thread_barrier.cpp

```diff
@@ -21,13 +21,13 @@ class CountBarriers : public IRVisitor {
     }
 };
 
-class CheckBarrierCount : public IRMutator {
+class CheckBarrierCount : public IRMutator2 {
     int correct;
 public:
     CheckBarrierCount(int correct) : correct(correct) {}
-    using IRMutator::mutate;
+    using IRMutator2::mutate;
 
-    Stmt mutate(Stmt s) {
+    Stmt mutate(const Stmt &s) override {
```

Comment:
Interesting. I think the old version of this was buggy

Reply:
Yep.